src/async-wrap.cc

```diff
@@ -62,6 +58,8 @@ static void Initialize(Handle<Object> target,
   HandleScope scope(isolate);
 
   NODE_SET_METHOD(target, "setupHooks", SetupHooks);
```

Comment:
I would do the same for SetupHooks for consistency.  That can wait for another PR though.